arbiter/server.go

```diff
@@ -182,22 +182,26 @@ func (s *Server) Run() error {
 
 	var syncErr error
 
+	syncCtx, syncCancel := context.WithCancel(ctx)
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
-		syncErr = syncBinlogs(s.kafkaReader.Messages(), s.load)
+		syncErr = syncBinlogs(syncCtx, s.kafkaReader.Messages(), s.load)
 		if syncErr != nil {
 			s.Close()
 		}
 	}()
 
 	err := s.load.Run()
 	if err != nil {
+		syncCancel()
 		s.Close()
 	}
 
 	wg.Wait()
 
+	// to pass go check
```

Comment:
address the comment, we don't write code for static check tools... the problem is why we need to cancel here?
just remove the comment is better to add a "to pass go check"